sphinx/ext/napoleon/__init__.py

```diff
@@ -236,6 +237,9 @@ def __unicode__(self):
 
             :returns: *bool* -- True if successful, False otherwise
 
+    napoleon_type_aliases : :obj:`dict` (Defaults to None)
+        Add a mapping of strings to string, translating types in numpy style docstrings.
```

Comment:
We have to mention this configuration only works when `napoleon_use_param` is True.

Reply:
In addition, please add a document for this into `doc/usage/extensions/napoleon.rst`.

Reply:
sure